src/Collection/CollectionInterface.php

```diff
@@ -95,6 +95,14 @@ public function filter(?callable $callback = null): CollectionInterface;
      */
     public function reject(callable $callback): CollectionInterface;
 
+    /**
+     * Returns unique elements
+     *
+     * @param string|null $key
+     * @return \Cake\Collection\CollectionInterface
+     */
+    public function unique(?string $key = null): CollectionInterface;
```

Comment:
Changing interfaces in minor releases is something we generally avoid as it causes fatal errors for other implementations. I think we've relaxed that for `CollectionInterface` in the past as there aren't any other known implementations.

Reply:
I agree when there're real changes. For example, one of the public methods was removed and return type was changed.

But what can go wrong when we add absolutely new method?